### ISSUE TITLE
if the background color is set to Colors.transparent, a weird issue arises.

### ISSUE THREAD

**Issue by tanzilzubair** [LanarsInc/animated-bottom-navigation-bar-flutter#2]
<img width="338" alt="Screen Shot 2020-07-09 at 5 41 04 PM" src="https://user-images.githubusercontent.com/47922073/87035593-5f72a880-c20b-11ea-94c9-89c91065156b.png">

**Comment by vizhan-lanars** [comment]
@tanzilzubair 
Can't reproduce.
Could you be more specific? 
Closed for now. But feel free to reopen, if needed.

**Comment by vizhan-lanars** [comment]
Code to reproduce
```
  @override
  Widget build(BuildContext context) {
    return Theme(
      data: ThemeData.dark(),
      child: Scaffold(
        extendBody: true,
        body: Container(color: Colors.red),
        floatingActionButton: FloatingActionButton(
          elevation: 8,
          child: Icon(Icons.call),
          onPressed: () {},
        ),
        bottomNavigationBar: Container(
          child: AnimatedBottomNavigationBar.builder(
            itemCount: iconList.length,
            tabBuilder: (int index, bool isActive) {
              final color = isActive ? Colors.green : Colors.white;
              return Column(
                mainAxisSize: MainAxisSize.min,
                mainAxisAlignment: MainAxisAlignment.center,
                children: [
                  Icon(
                    Icons.stream,
                    size: 24,
                    color: color,
                  ),
                  const SizedBox(height: 4),
                  Padding(
                    padding: const EdgeInsets.symmetric(horizontal: 8),
                    child: AutoSizeText(
                      "$index",
                      maxLines: 1,
                      style: TextStyle(color: color),
                      group: autoSizeGroup,
                    ),
                  )
                ],
              );
            },
            backgroundColor: Colors.black54,
            activeIndex: Random().nextInt(4),
            splashColor: Colors.green,
            notchAndCornersAnimation: animation,
            splashSpeedInMilliseconds: 300,
            notchSmoothness: NotchSmoothness.defaultEdge,
            gapLocation: GapLocation.none,
            leftCornerRadius: 32,
            rightCornerRadius: 32,
            onTap: (i) {
              setState(() {});
            },
          ),
        ),
      ),
    );
  }
  ```

**Comment by vizhan-lanars** [comment]
Fixed in version 0.3.1
